DependencyInjection/eZDemoCompilerPass.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+
+namespace EzSystems\DemoBundle\DependencyInjection;
+
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\Reference;
+
+class eZDemoCompilerPass implements CompilerPassInterface
```

Comment:
Currently I don't know any better idea to plug into PrivacyCookieBundle and override existing banner settings.